dv/cs_registers/tb_cs_registers.core

```diff
@@ -124,3 +124,5 @@ targets:
           - '-CFLAGS "-std=c++14 -Wall -DTOPLEVEL_NAME=tb_cs_registers -DVM_TRACE_FMT_FST -g"'
           - '-LDFLAGS "-pthread -lutil -lelf"'
           - "-Wall"
+          # Only a subset is test, print messages but do not fail with unsed warnings.
+          - '-Wno-fatal'
```

Comment:
How about only disabling the unused warnings (`-Wno-UNUSED`)?

Reply:
Because then the warning is not printed. I thought it would be better to see the warning and build it.
The "optimal" would be to disable the warning from being an error and still get the message.

Reply:
Ah I see. In that case let's slightly reword the comment to "Do not fail on (style) issues, only warn about them."

Reply:
Thanks, now reading the comment again I see that it is not only vague but also has an error.